waku/node/waku_node.nim

```diff
@@ -223,62 +236,104 @@ proc registerRelayDefaultHandler(node: WakuNode, topic: PubsubTopic) =
     await filterHandler(topic, msg)
     await archiveHandler(topic, msg)
 
-  node.wakuRelay.subscribe(topic, defaultHandler)
+  discard node.wakuRelay.subscribe(topic, defaultHandler)
 
-
-proc subscribe*(node: WakuNode, topic: PubsubTopic) =
+proc subscribe*(node: WakuNode, subscription: SubscriptionEvent) =
+  ## Subscribe to a pubsub topic without a handler.
+  ## Adds default handler to push messages to FILTER & STORE
+  
```

Comment:
This `proc` duplicates `subscribe` logic. Commented this below, but why not have this simply redirect to the more complete `subscribe` (with handler) below?

```suggestion
proc subscribe*(node: WakuNode, subscription: SubscriptionEvent) =
  ## Subscribe to a pubsub topic without a handler.
  ## Adds default handler to push messages to FILTER & STORE
  node.subscribe(subscription, none(WakuRelayHandler))
```

Of course, this implies moving this proc and changing the other one to accept an optional handler.

Reply:
OR use default handler of none